sample/BlobTrigger-CSharp/run.csx

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host;
+
+public static void Run(string id, TraceWriter log)
```

Comment:
See the Node.js BlobTrigger example. I'd expect your function here to take a WorkItem POCO which includes the id as well as other properties.